packages/core/models/notices/TokenNoticeModel.js

```diff
@@ -52,10 +50,12 @@ export default class TokenNoticeModel extends abstractNoticeModel({
   }
 
   message () {
-    const message = `notices.settings.erc20.tokens.${this.get('status')}`
-    return I18n.t(message, {
+    const value = `notices.settings.erc20.tokens.${this.get('status')}`
+
+    return {
+      value,
```

Comment:
Can be simplified like
`value: `notices.settings.erc20.tokens.${this.get('status')}``